thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/tools/CloneFunctionAndAutoTuneAlertFilterTool.java

```diff
@@ -0,0 +1,141 @@
+package com.linkedin.thirdeye.tools;
+
+
+import com.linkedin.thirdeye.anomaly.utils.DetectionResourceHttpUtils;
+import com.linkedin.thirdeye.anomaly.utils.OnboardResourceHttpUtils;
+import com.linkedin.thirdeye.datalayer.bao.AnomalyFunctionManager;
+import com.linkedin.thirdeye.datalayer.dto.AnomalyFunctionDTO;
+import com.linkedin.thirdeye.datalayer.util.DaoProviderUtil;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Utility class to support fetching results from autotune endpoint, fetching evaluation results from evaluation endpoint
+ * Also provide ways to write map to csv, read from csv to Map of Strings
+ */
+public class CloneFunctionAndAutoTuneAlertFilterTool {
+  private static final Logger LOG = LoggerFactory.getLogger(CloneFunctionAndAutoTuneAlertFilterTool.class);
+  public static AnomalyFunctionManager anomalyFunctionDAO;
+
+  private static final int DASHBOARD_PORT = 1426;
+  private static final int APPLICATION_PORT = 1867;
+  private static final String LOCALHOST = "localhost";
```

Comment:
Should we read these variables from the configuration files, e.g., dashboard.yml? When is this tool going to be used?